pkg/kubectl/cmd/drain.go

```diff
@@ -476,6 +476,13 @@ func (o *DrainOptions) getPodsForDeletion(nodeInfo *resource.Info) (pods []corev
 			if f != nil {
 				fs[f.string] = append(fs[f.string], pod.Name)
```

Comment:
Wow, terse (read unhelpful) names make review of a pull like this a real pain.  What are `w`, `f`, `ws`, and `fs`?

Ignore @smarterclayton and make variable names that tell little stories in them. :)  Or just give me a translation.

Reply:
go standard is that loop vars and core things are single letter, but normal use structs and other types should be one or two words.  This is too short.

Reply:
Sorry, loop vars and local object references (`func (a *Assembler) ...`) are one letter